### DISCUSSION TITLE
Can I install Coolify on an x86 architecture?

### DISCUSSION THREAD

**Discussion by mhmdsalahsebai** [coollabsio/coolify#3948] [Q&A] [answered]
I want to deploy a Next.js application with a MySQL database using Coolify. The instance I plan to use is based on an x86 architecture.

![Screenshot from 2024-10-18 14-44-06](https://github.com/user-attachments/assets/18ffcb0b-0e6e-4d3a-8937-458cb20f3e24)

**Comment by peaklabs-dev** [comment, marked as answer]
This should work perfectely, and you even have plenty of RAM. As long as your x86 system is running a 64bit OS like Ubuntu, this will work.

**Comment by mhmdsalahsebai** [comment]
Thank you!
